app/controllers/api/v4/drugs_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Api::V4::DrugsController < Api::V4::SyncController
+  skip_before_action :current_user_present?, only: [:sync_to_user]
+  skip_before_action :validate_sync_approval_status_allowed, only: [:sync_to_user]
+  skip_before_action :authenticate, only: [:sync_to_user]
+  skip_before_action :validate_facility, only: [:sync_to_user]
+  skip_before_action :validate_current_facility_belongs_to_users_facility_group, only: [:sync_to_user]
+
+  def sync_to_user
+    __sync_to_user__("drugs")
+  end
+
+  private
+
+  def current_facility_records
+    []
+  end
+
+  def other_facility_records
+    time(__method__) do
+      Drug.all
```

Comment:
Need to handle limit and updated_at filtering here

Reply:
If we return all the drugs every time, when does the app stop syncing?

Reply:
We're relying on an incidental factor here: The app defaults to a batch size of 1000, and there are only ~130 drugs. The app will assume sync has completed when it gets back < 1000 records for a sync request.

Potential failure scenarios:
- The Drug list grows to be > 1000 records, and the app either crashes when it gets back > 1000 records, or gets stuck in a sync loop as it keeps receiving the same records over and over. However, this list is very unlikely to grow so large before we move to an AR backed model, with the usual sync semantics. 
- The app changes its batch size to ~100 records, and runs into the same problem ^. We'll need to coordinate with the client to ensure this doesn't happen in the near future.

The reasons for returning all drugs here instead of using the simulated `updated_at` timestamps are:
- For a small batch size, the above problem still exists, since all the drug records share the same timestamp anyway.
- git (sometimes) rewrites the entire file, so the `created_at` and `updated_at` timestamps aren't always reliable. Given that the `updated_at` semantics don't reliably work, we probably shouldn't pretend that they do, and make this behaviour explicit instead by returning all drugs.

Reply:
Can we manually set a `VERSION` in the model and cursor based on that? We can send it in the process token, seems straightforward than using an `updated_at` at all. 

> We'll need to coordinate with the client to ensure this doesn't happen in the near future.

This is a little worrisome since in the bad case it causes sync loops for everyone. The limit is not set in stone, it's just a firebase config I think. If it's not much effort let's cursor it?

Reply:
This might be worth addressing if it takes about half a day to fix. Else, we can choose to accept the tradeoff for now and switch to active record when the time comes. Wdyt?

Reply:
Sounds good. 👍🏼

Reply:
Thought about this a bit more, I've decided to leave it as-is for now.

- Don't want to reinvent the wheel to implement sync-logic for a temporary implementation
- We've not actually changed the sync batch size for a long time, don't foresee it changing drastically (x10 reduction)
- We could fix it by just changing this to an AR backed model if necessary

Will make sure we test this in sandbox and demo envs so we don't break sync at launch